packages/editor/src/components/post-actions/index.js

```diff
@@ -21,27 +21,48 @@ import { usePostActions } from './actions';
 const { Menu, kebabCase } = unlock( componentsPrivateApis );
 
 function useEditedEntityRecordsWithPermissions( postType, postIds ) {
-	const { items, permissions } = useSelect(
+	const { items, entityConfig } = useSelect(
 		( select ) => {
-			const { getEditedEntityRecord, getEntityRecordPermissions } =
-				unlock( select( coreStore ) );
+			const { getEntityConfig } = select( coreStore );
+			const { getEditedEntityRecords } = unlock( select( coreStore ) );
+			const entityRecords = getEditedEntityRecords(
+				'postType',
+				postType,
+				postIds
```

Comment:
At the start of `PostActions`, there's a `useMemo` call:
```js
	const _postIds = useMemo( () => {
		if ( Array.isArray( postId ) ) {
			return postId;
		}
		return postId ? [ postId ] : [];
	}, [ postId ] );
```

I think that's unnecessary overhead, the coercion to an array can be performed in this `useSelect`. From what I could tell `useEditedEntityRecordsWithPermissions` is the only place `_postIds` is used, so I think it's a good idea to move that code inside this hook.